### PR TITLE
NXP-29372: add more info to package view

### PR DESCRIPTION
**Pull request by atchertchian**
Creating draft PR waiting for additional changes and feedback on preview instance from product/support

### CODE REVIEW

**Comment by atchertchian** [comment]
Will rebase to avoid hitting random ftest issues

**Comment by atchertchian** [comment]
Functional tests show that an error is visible in logs when the package references bundles not present in the distribution (can happen with partial distribs) --> will need to refine commits to avoid that

File: modules/nuxeo-apidoc-webengine/src/main/resources/skin/views/package/default.ftl [hunk 1]
```diff
@@ -76,6 +63,67 @@
...
+      </ul>
+    </div>
+  </#if>
+
+  <#if extensionpoints?size gt 0>
+    <h2>Extension Points</h2>
+    <div id="extensionpoints">
+      <ul>
+        <#list extensionpoints as xp>
+        <li><a class="extensionpoints" href="${Root.path}/${distId}/viewExtensionPoint/${xp.id}">${xp.id}</a></li>
```
Comment (atchertchian) [inline modules/nuxeo-apidoc-webengine/src/main/resources/skin/views/package/default.ftl:110]: CSS class should be "extensions"

**Comment by atchertchian** [comment]
Now that preview is visible, @tmartins seems happy with the changes.
Can i get reviews to be able to merge please? (will rebase #12 on master after that)

File: modules/nuxeo-apidoc-core/src/main/java/org/nuxeo/apidoc/introspection/ServerInfo.java [hunk 2]
```diff
@@ -536,7 +537,15 @@ public static ServerInfo build(String name, String version) {
             } else {
                 bi = server.bundles.get(bundle.getSymbolicName());
             }
-            bi.setDeploymentOrder(bindex++);
+            List<ComponentInfo> components = bi.getComponents();
+            OptionalLong min = components.stream().mapToLong(ComponentInfo::getRegistrationOrder).min();
+            if (min.isPresent()) {
+                bi.setMinRegistrationOrder(min.getAsLong());
+            }
```
Comment (efge) [inline modules/nuxeo-apidoc-core/src/main/java/org/nuxeo/apidoc/introspection/ServerInfo.java:544]: Could be `components.stream().mapToLong(ComponentInfo::getRegistrationOrder).min().ifPresent(min -> bi.setMinRegistrationOrder(min))`

**Pull request merged**